src/main/java/com/epam/task6/logic/search/factory/CreateSearch.java

```diff
@@ -0,0 +1,8 @@
+package com.epam.task6.logic.search.factory;
+
+import com.epam.task6.logic.Field;
+import com.epam.task6.logic.search.SearchBook;
+
+public interface CreateSearch {
```

Comment:
naming is a bit weird, SearchBook -> SearchSpecification (a noun) ; CreateSearch -> SearchSpecificationFactory